bee-network/bee-autopeering/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "bee-autopeering"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+description = """
+Allows peers in the same IOTA network to automatically discover each other.
+"""
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = ["iota", "bee", "framework", "network", "autopeering"]
+homepage = "https://www.iota.org"
+
+[dependencies]
+async-trait = { version = "0.1.51", default-features = false }
+base64 = { version = "0.13.0", default-features = false, features = [ "alloc" ] }
+bincode = { version = "1.3.3", default-features = false }
+bs58 = { version = "0.4", default-features = false, features = [ "alloc" ] }
+bytes = { version = "1.0", default-features = false }
+hash32 = { version = "0.2.1", default-features = false }
+hex = { version = "0.4.3", default-features = false }
+iota-crypto = { version = "0.9.1", default-features = false, features = [ "ed25519", "random", "sha" ] }
+libp2p-core = { version = "0.29.0", default-features = false }
+log = { version = "0.4", default-features = false }
+num = { version = "0.4.0", default-features = false }
+num-derive = { version = "0.3.3", default-features = false  }
+num-traits = { version = "0.2.14" , default-features = false }
+priority-queue = { version = "1.2.0" , default-features = false }
+prost = { version = "0.8", default-features = false, features = [ "std" ] }
+rand = { version = "0.8.4" , default-features = false }
+ring = { version = "0.16.20" , default-features = false }
+serde = { version = "1.0", default-features = false , features = [ "derive" ] }
+sled = { version = "0.34.7", default-features = false  }
+thiserror = { version = "1.0.29", default-features = false  }
+time = { version = "0.3.3", default-features = false, features = [ "std" ]  }
+tokio = { version = "1.11", default-features = false , features = [ "macros", "rt", "time", "net" ] }
+tokio-stream = { version = "0.1", default-features = false  }
+
+[dev-dependencies]
+fern = "0.6"
```

Comment:
We also put put `default-features = false` in the other crates, if it can reduce compilation time for tests why not (mostly for CI)

Reply:
yea, makes sense

Reply:
done